src/main/java/gg/playit/minecraft/utils/Logger.java

```diff
@@ -0,0 +1,35 @@
+package gg.playit.minecraft.utils;
+
+import gg.playit.minecraft.PlayitBukkit;
+import org.slf4j.LoggerFactory;
+
+public class Logger {
+    private static final org.slf4j.Logger LOGGER = LoggerFactory.getLogger("playit-minecraft-plugin");
+    public final String className;
+
+    public Logger(String className) {
+        this.className = className;
+    }
+
+    public void info(String format, Object... data) {
+        LOGGER.info("[" + className + "] " + format, data);
+    }
+
+    public void debug(String format, Object... data) {
+        if (PlayitBukkit.hasDebugLogging()) {
+            LOGGER.debug("[" + className + "] " + format, data);
```

Comment:
most servers will not print debug logs, I think we should use LOGGER.info here so the log statements are printed. If hasDebugLogging is set to false we can use `LOGGER.debug` and it will use the server's overall settings.

Reply:
I think we just do

```
if (PlayitBukkit.hasDebugLogging()) {
   LOGGER.info("[" + className + "] " + format, data);
} else {
   LOGGER.debug("[" + className + "] " + format, data);
}
```

Reply:
> I think we just do
> 
> ```java
> if (PlayitBukkit.hasDebugLogging()) {
>    LOGGER.info("[" + className + "] " + format, data);
> } else {
>    LOGGER.debug("[" + className + "] " + format, data);
> }
> ```

Oh, if I were to use that approach, I would have to switch to Bukkit's logger. I already wrote something to use the `spigot.yml` file directly but I should probably switch to Bukkit's logger.